apps/price_pusher/src/logger.ts

```diff
@@ -0,0 +1,35 @@
+import pino from "pino";
+import { Logger, Level } from "pino";
+import { Logger as LoggerInterface } from "ts-log";
+export { Logger } from "pino";
+
+export type LogFormat = "json" | "pretty";
+
+export function createLogger(logLevel?: Level, logFormat?: LogFormat): Logger {
+  return pino({
+    level: logLevel ?? "info",
+    transport:
+      logFormat === "pretty"
+        ? {
+            target: "pino-pretty",
```

Comment:
I generally don't think `pino-pretty` should be installed directly into the code.  It's really designed to take the log stream and format it by piping and it's just as easy for any downstream consumer to pipe their log streams themselves if they want to, and the authors themselves [recommend against including it as a production dependency](https://github.com/pinojs/pino-pretty?tab=readme-ov-file#programmatic-integration).

I would instead remove the `logFormat` option and just document in the README or something how someone could use `pino-pretty` if they wanted pretty logs.

Reply:
Done. I removed it entirely and updated the readme.